src/Helper/Quote.php

```diff
@@ -64,7 +65,9 @@ protected function setShipping()
                                   ->setCollectShippingRates(true)
                                   ->collectShippingRates()
                                   ->getShippingRateByCode($methodName);
-        $this->quote->getShippingAddress()->setShippingMethod($rate ? $methodName : 'shopgate_fix');
+        $this->quote->getShippingAddress()
+                    ->setShippingMethod($rate ? $methodName : 'shopgate_fix')
+                    ->setData(OrderCondition::CLIENT_ATTRIBUTE, $this->sgBase->getClient()->getType());
```

Comment:
check for null of `getClient` as it will throw an exception (and I will have to mock these values forever in all my addOrder tests).